### PR TITLE
Added cart page

### PR DESCRIPTION
**Pull request by aiman-ansari**
Screenshot (57)

### CODE REVIEW

**Review by Debarshi95 — APPROVED** [review]
Looks good.

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5502
+}
```
Comment (manojsarna) [inline .vscode/settings.json:3]: can use gitignore for this

File: Cart/cart.html [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+                <div class="card flex box-shadow-light pd-1">
+                    <img class=" img-sm" src="/assest/adidas.jpg"/>
+                    <div class="flex-col card-body ml-1">
+                        <div class="flex gap-3">
+                            <div>
+                                <div class="card-title text-sm">American Tourister </div>
+                                <div class="card-description">Americn tourister bag </div>
+                            </div>
+                            <div>
+                                <i class="fa fa-trash fa-1x text-accent"></i>
```
Comment (manojsarna) [inline Cart/cart.html:44]: use webp instead jpg , for better performance and instant load time

**Review by manojsarna — APPROVED** [review]
Overall looks good!!

you can merge

**Review by j24m — APPROVED** [review]
Hey, UI looks good , I like the colors you have used, coming back to the code, it looks fine, 5/5 for the efforts. :)

**Pull request merged**